.kitchen.yml

```diff
@@ -1,15 +1,14 @@
 ---
 provisioner:
+  name: chef_zero
   data_bag_path: test/integration/data_bags
 
 suites:
   - name: default
+    attributes:
+      poise-python:
+        provider: system
```

Comment:
This attribute is actually set in the `base` role but i just wanted to add it here so testing worked correctly.